configs/samples/simpleusb.conf.sample

```diff
@@ -90,6 +90,10 @@
 
 ;duplex3 = 0                ; duplex 3 gain setting (0 to disable)
 
+;clipledgpio = 1            ; Enable ADC Clip Detect feature to use a GPIO output (0 to disable).
```

Comment:
I know it's commented out by default but maybe for consistency both configs/samples files should have it set to 0 here? Just to avoid confusion

Reply:
Per our earlier comments I updated it to disabled by default in the .c files, and it is consistent between the configs/rpt/ and .sample files. I think it makes sense to leave it disabled by default on existing nodes ie. during updates, but that for new node installs the default (for at least the simpleusb driver) should be enabled, because during a new install is when users will be reviewing the config settings and selecting the values that make sense for them (and setting up any GPIO events). If the setting were not enabled in a new install though, I think there is a tendency that with the people who are less proficient in setting audio levels are probably also more likely to be less proficient with other config settings, and thus never make use of a feature that could alert them to audio level issues.

Reply:
> Per our earlier comments I updated it to disabled by default in the .c files, and it is consistent between the configs/rpt/ and .sample files. I think it makes sense to leave it disabled by default on existing nodes ie. during updates, but that for new node installs the default (for at least the simpleusb driver) should be enabled, because during a new install is when users will be reviewing the config settings and selecting the values that make sense for them (and setting up any GPIO events). If the setting were not enabled in a new install though, I think there is a tendency that with the people who are less proficient in setting audio levels are probably also more likely to be less proficient with other config settings, and thus never make use of a feature that could alert them to audio level issues.

I agree with your logic, but that's what the rpt files are for. The samples files aren't supposed to be judgmental like that, they are really just documentation, hence why the difference is a bit confusing from that perspective.

Reply:
OK I think I understand what you're saying about the .sample files being more general to Asterisk and just documenting what configs are there rather than recommending specific values (correct me if I'm wrong). So I just updated that in configs/samples/simpleusb.conf.sample to 0, which now matches usbradio.conf.sample.